dm/dm.go

```diff
@@ -295,13 +325,20 @@ func describeType(args []string) {
 		usage()
 	}
 
-	tUrls := getDownloadURLs(args[1])
+	tUrls := getDownloadURLs(url.QueryEscape(args[1]))
 	if len(tUrls) == 0 {
 		panic(fmt.Errorf("Invalid type name, must be a template URL or in the form \"<type-name>:<version>\": %s", args[1]))
 	}
 
-	schemaUrl := tUrls[0] + ".schema"
-	fmt.Println(callHttp(schemaUrl, "GET", "get schema for type ("+tUrls[0]+")", nil))
+	if !strings.Contains(tUrls[0], ".prov") {
+		// It's not a chart, so grab the schema
+		path := fmt.Sprintf("registries/%s/download?file=%s.schema", *template_registry, url.QueryEscape(tUrls[0]))
```

Comment:
.py or .jinja
.py.schema file is then fetched since we tack on .schema to fetch it.